### PR TITLE
DEX-710: multi-value customField values serialized as one value/string

### PR DESCRIPTION
**Pull request by naknomum**
## Pull Request Overview

- Add test to confirm multi-value CustomFields (in EDM) is working
- A corresponding [update in EDM](https://github.com/WildMeOrg/Wildbook/commit/95f6aa88b83bc512e4dd505c1667b01d45dbe444) is required

### CODE REVIEW

File: tests/modules/sightings/resources/test_custom_fields.py [hunk 1]
```diff
@@ -108,16 +114,28 @@ def test_custom_fields_on_sighting(
...
+                cfd_multi_id: cfd_multi_value,
+            },
             # Only asserting that these fields exist
             'time': full_sighting.json['time'],
             'timeSpecificity': full_sighting.json['timeSpecificity'],
             'createdEDM': full_sighting.json['createdEDM'],
         }
     )
 
+    # for some reason the assert above *passes* even though cfd_multi_value is wrong (maybe the >=)
```
Comment (naknomum) [inline tests/modules/sightings/resources/test_custom_fields.py:128]: comment should say "even **when**".  that is to say, the above test passed when edm was returning the _wrong value_ for the multi-value.

**Review by colinwkingen — APPROVED** [review]
LGTM

**Pull request merged**